packages/beacon-node/src/network/network.ts

```diff
@@ -107,7 +107,6 @@ export class Network implements INetwork {
 
   private subscribedToCoreTopics = false;
   private connectedPeers = new Set<PeerIdStr>();
-  private regossipBlsChangesPromise: Promise<void> | null = null;
```

Comment:
this is unused becasue code is commented out

https://github.com/ChainSafe/lodestar/blob/408033e17f08a64416a6cbb74eaa85a74113edce/packages/beacon-node/src/network/core/networkCore.ts#L487-L500

seems related to https://github.com/ChainSafe/lodestar/pull/5229 but it's unclear ot me if this is relevant, should probably removed commented out code as well if it is not cc @wemeetagain or @twoeths as you were reviewers of that initial PR

Reply:
I deleted the commented code https://github.com/ChainSafe/lodestar/pull/7759/commits/e0d77e32d57dcd91cbf01cf506da1e8f430a5778